sale_order_expected_date/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Quartile (https://www.quartile.co)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+{
+    "name": "Sale Order Delivery Date",
+    "version": "18.0.1.0.0",
+    "author": "Quartile",
+    "website": "https://www.quartile.co",
+    "category": "Sale",
+    "license": "LGPL-3",
```

Comment:
Please change the license to AGPL.
```suggestion
    "license": "AGPL-3",
```